src/__tests__/wait-for.js

```diff
@@ -194,14 +194,14 @@ test('when a promise is returned, if that is not resolved within the timeout, th
   await sleep(5)
 
   expect((await waitForError).message).toMatchInlineSnapshot(`
-Timed out in waitFor.
-
-Ignored nodes: comments, <script />, <style />
```

Comment:
Can we keep the original formatting? "comments, scripts, style" looks like they all refer to the same thing while "scripts, style" specifically refers to elements.

Reply:
Should be fine. Alternative is another `.split().map().join()` which is probably too expensive.